tests/system/libraries/_chromeArgs.py

```diff
@@ -14,7 +14,12 @@ def getChromeArgs() -> str:
 	"""
 	return (
 		"chrome"  # Start Chrome
+		" --no-first-run"  # Don't show intro pages for the first run of chrome
 		" --force-renderer-accessibility"  # don't rely on chrome detecting a screen reader.
+		" --ash-no-nudges"  # Prevents Chrome from showing nudge messages.
+		" --browser-test"  # Enable browser test mode, helps reduce flakey tests

```

Comment:
Is there anymore info you can provide on how it reduces flaky tests? all in same process, or reduces UI clutter...?

Reply:
There's not much info from chrome here just the following:

>Tells whether the code is running browser tests (this changes the startup URL used by the content shell and also disables features that can make tests flaky [like monitoring of memory pressure])

https://peter.sh/experiments/chromium-command-line-switches/#browser-test